packages/alfa-scraper/test/scraper.spec.ts

```diff
@@ -1,10 +1,13 @@
 import * as url from "url";
 
+import { Node, Query } from "@siteimprove/alfa-dom";
 import { test } from "@siteimprove/alfa-test";
 import { Awaiter } from "../src";
 
 import { Scraper } from "../src/scraper";
 
+const { getElementDescendants } = Query;
+
 const fixture = `${url.pathToFileURL(__dirname).href}/fixture`;
```

Comment:
**Suggestion:** We should take the opportunity to use `path.join` instead, to handle Unix-Windows `/`-`\` war 🙄

Reply:
Yes, let's do that. Btw, calling `path.join` after having converted the base path to a file URL messed up the URL so the tests broke. Therefore I changed it to first do all the path joining and then at the end converting to a file URL and put it in a helper function.